src/pages/index.js

```diff
@@ -30,49 +32,126 @@ const profileDescriptionInput = document.querySelector(
 );
 
 /*=============================================
-=           Create Cards          =
+=            server request            =
 =============================================*/
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "3235a097-fe3d-431f-b680-2a5fa7b7897e",
+    "Content-Type": "application/json",
+  },
+});
+
+let cardList;
+let userInfo;
+
+api
+  .loadPageResults()
+  .then(([card, userData]) => {
+    cardList = new Section(
+      {
+        items: [...initialCards, ...card],
+        renderer: createCard,
+      },
+      ".cards__list"
+    );
+    cardList.renderItems();
+    userInfo = new UserInfo(
+      ".profile__title",
+      ".profile__description",
+      ".profile__image"
+    );
+    userInfo.setUserInfo({ title: userData.name, description: userData.about });
+    userInfo.setUserImage(userData.avatar);
+  })
+  .catch(console.error);
+
+/*=============================================
+=           Create Cards          =
+=============================================*/
+//fetchnewcard
 function createCard(cardData) {
   const cardElement = new Card(
     cardData,
     "#card-template",
-    handlePreviewPicture
+    handlePreviewPicture,
+    handleDeleteClick,
+    handleLikeClick
   );
   return cardElement.getView();
 }
 
 /*=============================================
-=            User Info            =
+=           profile Info            =
 =============================================*/
 
-const userInfo = new UserInfo(".profile__title", ".profile__description");
-
-function handleProfileEditSubmit(UserData) {
-  profilePopupForm.close();
-  userInfo.setUserInfo(UserData);
+function handleProfileEditSubmit(userData) {
+  profilePopupForm.setLoading(true);
+  api
+    .fetchEditProfile(userData)
+    .then((userData) => {
+      profilePopupForm.close(userData);
+    })
+    .catch((err) => console.error(err))
+    .finally(() => profilePopupForm.setLoading(false));
 }
 
 /*=============================================
-=            section            =
+=            profile image edit            =
 =============================================*/
 
-const cardList = new Section(
-  {
-    items: initialCards,
-    renderer: createCard,
-  },
-  ".cards__list"
+const profileImagePopupForm = new PopupWithForm(
+  "#picture-modal",
+  handleProfilePictureEdit
 );
-cardList.renderItems();
+profileImagePopupForm.setEventListeners();
 
-function handleAddCardEditSubmit(cardData) {
-  const name = cardData.title;
-  const link = cardData.url;
-  cardList.addItem({ name, link });
+const profileImageEditButton = document.querySelector(
+  ".profile-image__edit-button"
+);
+
+profileImageEditButton.addEventListener("click", (evt) => {
+  evt.preventDefault();
+  profileImagePopupForm.open();
   addFormValidator.disableButton();
```

Comment:
`addFormValidator.disableButton();` is for the card form rather than the avatar

You need to `disable` the button only after **resetting** the form when you have submitted a card otherwise users may close the modal by accident and they don’t expect that the button will be disabled on opening if the inputs have **valid** data which they haven’t submitted yet.